enigma/jwt/header.go

```diff
@@ -0,0 +1,20 @@
+package jwt
+
+// HeaderContext is the context for a jwt header.
+type Header struct {
```

Comment:
Header is now a struct because we can easily add functionality in future releases.